### ISSUE TITLE
ourworldoftext.com ruleset

### ISSUE THREAD

**Issue by SuperOP535** [EFForg/https-everywhere#17136]
[//]: # (Thank you for reporting an issue to HTTPS Everywhere.)
[//]: # (We welcome input from users on improving this project.)
[//]: #
[//]: # (Please help us by following this issue template.)
[//]: # (You can delete all blank lines and all lines starting)
[//]: # (with the comment marker, such as this one.)
[//]: #
[//]: # (Delete all but one of the following "Type" lines.)
[//]: # (Leave in the line that best describes the issue you are reporting.)
[//]: #
[//]: # (If you are submitting a new ruleset, please check the list at)
[//]: # (https://www.eff.org/https-everywhere/atlas/index.html and the open)
[//]: # (issues and pull requests to make sure it doesn't already exist.)

Type: new ruleset

[//]: # (If you are reporting a ruleset/website problem, include the top-level)
[//]: # (domain below. For example, if you want to report an issue about)
[//]: # ("one.example.com" and "two.example.com", then the line below should)
[//]: # (be:)
[//]: #
[//]: # (Domain: example.com)
[//]: #
[//]: # (Be sure to remove the parenthesis and comment marker.  If you are only)
[//]: # (reporting an issue about "one.example.com", then the line below should)
[//]: # (be:)
[//]: #
[//]: # (Domain: one.example.com)
[//]: #
[//]: # (Only include one top-level domain. If you have more than one top-level)
[//]: # (domain to report, such as both "example.com" and "example.org", open a)
[//]: # (new issue for each top-level domain.)

Domain: ourworldoftext.com

[//]: # (Include any other relevant information below. Thank you again for)
[//]: # (helping to improve HTTPS Everywhere.)

**Comment by SuperOP535** [comment]
@Bisaloo how much time does this take / should I make a PR instead?

**Comment by Bisaloo** [comment]
We don't have many contributors and many of them have their own priorities for this project. It would indeed be faster if you make a PR yourself.

You should find all the information you need in our [contributing guide](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md). Let me know if you have any questions and feel free to ping me to review your PR once it is ready.

**Comment by pipboy96** [comment]
Closed by #17358.
